### ISSUE TITLE
Cordova command requires extra flag on XCode 10

### ISSUE THREAD

**Issue by mesqueeb** [quasarframework/quasar-cli#215]
Hi,
As per the [cordova announcement](https://github.com/apache/cordova-ios/issues/407) cordova can only be initialised when you add the extra flag `--buildFlag='-UseModernBuildSystem=0'` to Cordova if you have XCode 10.

I tried to do this with quasar dev:
```
quasar dev -m cordova -T ios -t ios -e iPhone-8 --buildFlag='-UseModernBuildSystem=0'
```
But Quasar seems to not succesfully pick up this flag...

Without this flag [errors prevent the simulator](https://stackoverflow.com/questions/52584550/blank-app-fails-debug-build-info-plist-file-not-found) from being run.

**Comment by rstoenescu** [comment]
Try adding this to build.json as per cordova's recommendation:

```json
"ios": {
    "release": {
      "buildFlag": [
        "-UseModernBuildSystem=0"
      ]
    }
  }
```

**Comment by mesqueeb** [comment]
@rstoenescu There is no `build.json` file in the `src-cordova` folder and also not in the main quasar folder.
I also tried adding it under `cordova` inside quasar.conf but this also did not work.
Any advice?

**Comment by rstoenescu** [comment]
Yes, by default there isn't such a file. But you create it yourself in /src-cordova

**Comment by mesqueeb** [comment]
On the android area:

In order for my app to be signed each build & during dev I need to add additional params to cordova's `build.json` file.

[Cordova docs on android app signing](https://cordova.apache.org/docs/en/latest/guide/platforms/android/index.html#signing-an-app)

So I guess we'll change this thread to "how to make Quasar use Cordova build.json file".

**Comment by mesqueeb** [comment]
My iOS problem was solved as well:

Just for anyone with a similar problem and want to **get it to work with the build.json**

At first I tried what I have written in the beginning of this thread, but it didn't work:
`build.json`
```
  "ios": {
    "release": {
      "buildFlag": [
        "-UseModernBuildSystem=0"
      ]
    }
  },
```
Then I realised I have to add "debug" as well.
### Solution:
`build.json`
```js
  "ios": {
    "debug": {
      "buildFlag": [
        "-UseModernBuildSystem=0"
      ]
    },
    "release": {
      "buildFlag": [
        "-UseModernBuildSystem=0"
      ]
    }
  },
```
Just thought I'd leave it here for future reference.

**Comment by mesqueeb** [comment]
Also important to note that when you use this method, you need to select the "Legacy Build System" in Xcode before you archive your app, because Cordova is not yet compatible with the new build system.

Go to `file` > `project settings` (or workspace settings) and then select "Legacy Build System"
